hoomd_organics/base/system.py

```diff
@@ -261,6 +265,19 @@ def _remove_hydrogens(self):
         if len(hydrogens) > 0:
             self.gmso_system = from_parmed(parmed_struc)
 
+    def _scale_charges(self):
+        """"""
+        charges = np.array(
+            [
+                site.charge if site.charge else 0
+                for site in self.gmso_system.sites
+            ]
+        )
+        net_charge = sum(charges)
+        abs_charge = sum(abs(charges))
+        for site in self.gmso_system.sites:
+            site.charge -= abs(site.charge) * (net_charge / abs_charge)
```

Comment:
probably need to take care of division by zero error for cases where `abs_charge=0`

Reply:
And `abs(site.charge)` would also cause problem if `site.charge=None`.

I think one way to fix this would be to have a for loop in `_convert_to_gmso()`  that checks the `site.charge` and if it's None sets to 0 by default.

Reply:
Good catch, I'll fix this after we merge your PR with addl. unit tests.